hummingbot/core/event/events.py

```diff
@@ -154,9 +155,11 @@ class BuyOrderCompletedEvent(NamedTuple):
     quote_asset_amount: Decimal
     fee_amount: Decimal
     order_type: OrderType
+    exchange_order_id: Optional[str] = ""
```

Comment:
Shouldn't `Optional[str]` be assigned None value initially?

Reply:
Right. Refactored.